dcpy/connectors/edm/recipes.py

```diff
@@ -70,7 +70,7 @@ def import_recipe(
     recipe_version = config["dataset"]["version"]
     sql_script_path = fetch_sql(recipe_name, recipe_version, local_library_dir)
 
-    postgres.exec_file_via_shell(postgres.BUILD_ENGINE_RAW, sql_script_path)
+    postgres.execute_file_via_shell(postgres.BUILD_ENGINE_RAW, sql_script_path)
```

Comment:
Oops. Also, if you saw, I went ahead and just deleted the part of this function doing anything with the source versions table - figured since this isn't even being used yet, we won't plan around something we're going to phase out